libbeat/dashboards/kibana_loader.go

```diff
@@ -0,0 +1,97 @@
+package dashboards
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/url"
+
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/logp"
+	"github.com/elastic/beats/libbeat/setup/kibana"
+)
+
+var importAPI = "/api/kibana/dashboards/import"
+var exportAPI = "/api/kibana/dashboards/export"
+
+type KibanaLoader struct {
+	client       *kibana.Client
+	config       *DashboardsConfig
+	version      string
+	msgOutputter *MessageOutputter
+}
+
+func NewKibanaLoader(cfg *common.Config, dashboardsConfig *DashboardsConfig, msgOutputter *MessageOutputter) (*KibanaLoader, error) {
+
+	if cfg == nil {
+		cfg = common.NewConfig()
+		err := cfg.SetBool("enabled", -1, true)
```

Comment:
"enabled" is forced to true here, but I didn't see where it's actually checked. We should make sure that disabling with `enabled: false` works.